packages/eslint-config-vivy/index.js

```diff
@@ -31,4 +31,14 @@ module.exports = {
       jsx: true,
     },
   },
+  overrides: [
+    {
+      files: ["**/*.ts", "**/*.tsx"],
+      rules: {
+        "no-unused-vars": "off",
+        "no-undef": "off",
```

Comment:
Can be helpful in PRs to give more details about why you disabled these based on your research, otherwise the reviewer just has to do the same. 

I guess its the advice from here: https://github.com/typescript-eslint/typescript-eslint/issues/342#issuecomment-484739065 ?

Reply:
I thought I did with the current description :D

Reply:
I guess what was missing is that these are covered by TSC so there should be no discernable differences